melvs/melvsboot/src/main/java/com/melvs/melvsboot/Controller.java

```diff
@@ -0,0 +1,56 @@
+package com.melvs.melvsboot;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+//controller rest mapping
+@RestController
+public class Controller {
+
+//means needs another dependencies to map
+    @Autowired
```

Comment:
looks like this @Autowired is connected to the @Service annotation you added in your Business Service class. Am I right?

Reply:
yes, @service is connected to @autowired cause its calling the @service for its dependency

Reply:
Instead of using autowired, for classes marked as Component, and Service, you can inject them using constructor. There are performance issues found using Autowired used in private variables.